reana_workflow_controller/consumer.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2018 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation; either version 2 of the License, or (at your option) any later
+# version.
+#
+# REANA is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# REANA; if not, write to the Free Software Foundation, Inc., 59 Temple Place,
+# Suite 330, Boston, MA 02111-1307, USA.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization or
+# submit itself to any jurisdiction.
+
+"""REANA Workflow Controller MQ Consumer."""
+
+import json
+
+import pika
+from reana_commons.database import Session
+from reana_commons.models import WorkflowStatus
+
+from .config import (BROKER_PASS, BROKER_PORT, BROKER_URL, BROKER_USER,
+                     STATUS_QUEUE)
+from .tasks import (_update_job_progress, _update_run_progress,
+                    _update_workflow_status)
+
+
+class Consumer:
```

Comment:
What do you think about renaming to `JobStatusConsumer` to better describe what it does?

Reply:
We could also think about container names `[rest job-status-consumer]` to be something like `[rest-api-server job-status-consumer]`.  I.e. both components being named "-er" in the same noun way. What about merging and amending all those names in another PR while @dinosk is away?